cvat/apps/auto_segmentation/views.py

```diff
@@ -211,37 +248,58 @@ def create(request, tid):
         db_labels = db_task.label_set.prefetch_related('attributespec_set').all()
         db_labels = {db_label.id:db_label.name for db_label in db_labels}
 
-        # COCO Labels
-        auto_segmentation_labels = { "BG": 0,
-            "person": 1, "bicycle": 2, "car": 3, "motorcycle": 4, "airplane": 5,
-            "bus": 6, "train": 7, "truck": 8, "boat": 9, "traffic_light": 10,
-            "fire_hydrant": 11, "stop_sign": 12, "parking_meter": 13, "bench": 14,
-            "bird": 15, "cat": 16, "dog": 17, "horse": 18, "sheep": 19, "cow": 20,
-            "elephant": 21, "bear": 22, "zebra": 23, "giraffe": 24, "backpack": 25,
-            "umbrella": 26, "handbag": 27, "tie": 28, "suitcase": 29, "frisbee": 30,
-            "skis": 31, "snowboard": 32, "sports_ball": 33, "kite": 34, "baseball_bat": 35,
-            "baseball_glove": 36, "skateboard": 37, "surfboard": 38, "tennis_racket": 39,
-            "bottle": 40, "wine_glass": 41, "cup": 42, "fork": 43, "knife": 44, "spoon": 45,
-            "bowl": 46, "banana": 47, "apple": 48, "sandwich": 49, "orange": 50, "broccoli": 51,
-            "carrot": 52, "hot_dog": 53, "pizza": 54, "donut": 55, "cake": 56, "chair": 57,
-            "couch": 58, "potted_plant": 59, "bed": 60, "dining_table": 61, "toilet": 62,
-            "tv": 63, "laptop": 64, "mouse": 65, "remote": 66, "keyboard": 67, "cell_phone": 68,
-            "microwave": 69, "oven": 70, "toaster": 71, "sink": 72, "refrigerator": 73,
-            "book": 74, "clock": 75, "vase": 76, "scissors": 77, "teddy_bear": 78, "hair_drier": 79,
-            "toothbrush": 80
-            }
-
-        labels_mapping = {}
-        for key, labels in db_labels.items():
-            if labels in auto_segmentation_labels.keys():
-                labels_mapping[auto_segmentation_labels[labels]] = key
+        if int(mid) == 989898:
```

Comment:
Please use django fixtures or post migrate signals to add proper records to the database. It will allow to not use a magic numbers.

Reply:
Thanks. I need to check that. I will get back to you. Just to summarize what we do here, we assigned a specific unique number to the default models because we had `int` in the request.